cmd/cmd.go

```diff
@@ -0,0 +1,176 @@
+// Package cmd - package
+package cmd
+
+import (
+	"errors"
+	"fmt"
+	"os"
+	"strconv"
+
+	"github/Asac2142/go-cli-app/file"
+	"github/Asac2142/go-cli-app/task"
+)
+
+const (
+	add            string = "add"
+	update         string = "update"
+	deleteTask     string = "delete"
+	markInProgress string = "mark-in-progress"
+	markDone       string = "mark-done"
+	list           string = "list"
+)
+
+// HandleTrackerCLI - Handles user's input
```

Comment:
General comment format (at least for public entries and package itself):
```
// <FuncName> <natural language sentence with the full stop in the end>
```
so it should be
```
// HandleTrackerCLI handles user's input.
```
or maybe
```
// HandleTrackerCLI handles command arguments.
```